databricks/koalas/window.py

```diff
@@ -80,6 +80,34 @@ def __getattr__(self, item: str) -> Any:
                 return partial(property_or_func, self)
         raise AttributeError(item)
 
+    def _apply_as_series_or_frame(self, func):
```

Comment:
Should I maybe separate `Expanding` into `SeriesExpandng` and `FrameExpanding` .. ? not sure yet.